docs/ref/validators.txt

```diff
@@ -154,7 +154,8 @@ to, or in lieu of custom ``field.clean()`` methods.
     an error code of ``'invalid'`` if it doesn't.
 
     Loopback addresses and reserved IP spaces are considered valid. Literal
-    IPv6 addresses (:rfc:`2732`) and unicode domains are both supported.
+    IPv6 addresses (:rfc:`2732#section-2`) and unicode domains are both
+    supported.
```

Comment:
This RFC is obsolete I think we should use `rfc3986#section-3.2.2`.

Reply:
I've updated the RFC number (and paragraph) as suggested.

I've also checked to see if RFC 2732 was referenced anywhere else but that doesn't appear to be the case.